main.py

```diff
@@ -0,0 +1,27 @@
+import os
+from dotenv import load_dotenv
+from doctors import Doctors
+
+# Load environment variables from .env file
+load_dotenv()
+
+# Define the URLs
+base_url = "https://production.api-annuaire-sante.fr/professionnel_de_santes?libelleSavoirFaire[]=Médecine Générale"
+login_url = "https://production.api-annuaire-sante.fr/login_check"
+
+# Define the credentials
+email = os.getenv("EMAIL")
+password = os.getenv("PASSWORD")
+
+if not email or not password:
+    raise ValueError("EMAIL and PASSWORD environment variables must be set")
+
+# Create an instance of the Doctors class
+doctors = Doctors(base_url, login_url, email, password)
+
+# Get the doctors
+try:
+    doctors_list = doctors.get_doctors()
+    print(doctors_list)
+except Exception as e:
+    print(e)
```

Comment:
I would let it raise, it's a top level module and we don't do any fancy logging so it's more code and not more value